sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAsyncClient.java

```diff
@@ -60,23 +127,24 @@
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<AttestationResponse> attestOpenEnclave(AttestOpenEnclaveRequest request) {
-        return serviceClient.attestOpenEnclaveAsync(request.toGenerated())
-            .map(AttestationResponse::fromGenerated);
+        return withContext(context -> attestOpenEnclaveWithResponse(request, context))
```

Comment:
>attestOpenEnclaveWithResponse [](http://example.com/codeflow?start=38&length=29)

Move to FluxUtil::toMono - across the board :)

Reply:
I understand why I'd use FluxUtil::toMono when converting a Response<T> to T, but I don't understand why I would use it in APIs which return a Response<T>

Reply:
Also, I don't see how this works for APIs which require a transform from an internal model type to a public model type. Do you have examples of using toMono to convert between non response types?

Reply:
This mainly guards against scenarios where response.getValue() can be void. If you are certain that is not the case - what you have will behave exactly the same way as the Core helper method.

Reply:
Ok, makes sense to me. I'm going to have to do a bit of a cleanup pass as I move forward, I'll keep this in mind.